qcodes/plots/pyqtgraph.py

```diff
@@ -348,23 +348,42 @@ def _update_labels(self, subplot_object, config):
         """
         Updates x and y labels, by default tries to extract label from
         the DataArray objects located in the trace config. Custom labels
-        can be specified the **kwargs "xlabel" and "ylabel"
+        can be specified the **kwargs "xlabel" and "ylabel". Custom units
+        can be specified using the kwargs xunit, ylabel
         """
         for axletter, side in (('x', 'bottom'), ('y', 'left')):
             ax = subplot_object.getAxis(side)
+            # danger: 🍝
```

Comment:
why danger? I'd say that if someone wants to overwrite a label (to e.g., more clearly distinguish two voltages) that is intended behaviour.